src/bridge/secrets/xpro/secrets.ci.yaml

```diff
@@ -0,0 +1,139 @@
+---
+google_analytics:
```

Comment:
Thanks but what makes you say that? I want to see the source of truth you're seeing so I'm not just running around in the dark poking things with a stick :)

Reply:
Because of the level of nesting in the YAML document. There are two top-level keys in the document right now, `google_analytics` and `misc`. Everything else is nested under those keys.

Reply:
I just pushed a commit with the nesting changed to have more of the keys be top-level so that they will iterate properly.